<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>658</width>
    <height>444</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 88, 88);

</string>
  </property>
  <property name="iconSize">
   <size>
    <width>45</width>
    <height>45</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="11">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>811</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="11" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="btn_minimize">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>-</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/User/Desktop/collapse.png</normaloff>C:/Users/User/Desktop/collapse.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_maximize">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/User/Desktop/expandv2.0.png</normaloff>C:/Users/User/Desktop/expandv2.0.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_close">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>C:/Users/User/Desktop/close.png</normaloff>C:/Users/User/Desktop/close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="13">
     <widget class="QTableView" name="playlistView">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(77, 77, 77);
</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="btn_add">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/add.png</normaloff>:/icon/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="12">
     <widget class="QLabel" name="currentTrack">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Courier New&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Your Track</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QToolButton" name="btn_previous">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/previous.png</normaloff>:/icon/previous.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>37</width>
        <height>45</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QToolButton" name="btn_play">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/play.png</normaloff>:/icon/play.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QToolButton" name="btn_stop">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/stop.png</normaloff>:/icon/stop.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QToolButton" name="btn_next">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/next.png</normaloff>:/icon/next.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>56</width>
        <height>78</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QCheckBox" name="Loop">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>21</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(159, 159, 159);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/loop.png</normaloff>:/icon/loop.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QCheckBox" name="Rand">
      <property name="minimumSize">
       <size>
        <width>45</width>
        <height>21</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>45</width>
        <height>21</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(159, 159, 159);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/icon/rand.png</normaloff>:/icon/rand.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>293</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="9">
     <widget class="QToolButton" name="btn_volume">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(88, 88, 88);</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="10" colspan="2">
     <widget class="QSlider" name="volume">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>22</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="12">
     <widget class="QLabel" name="currentvolume">
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>50</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="currentdura">
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0:00</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="5">
     <widget class="QSlider" name="dura">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QLabel" name="maxdura">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action">
   <property name="text">
    <string>Вихід</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
